lib/connect.js

```diff
@@ -414,7 +414,7 @@ function connect(token, options) {
  * You may pass these options to {@link connect} in order to override the
  * default behavior.
  * @typedef {object} ConnectOptions
- * @property {boolean|CreateLocalTrackOptions} [audio=true] - Whether or not to
+ * @property {boolean|CreateLocalAudioTrackOptions} [audio=true] - Whether or not to
```

Comment:
`boolean|CreateLocalTrackOptions|CreateLocalAudioTrackOptions`? Or else it won't match ts defs

Reply:
fixed the TS defs now.

Reply:
Same here, instead of updating the tsdef, we should update this instead to prevent a breaking change.

Reply:
addressed now.